telemetry/specs/telemetry_report.mdk

```diff
@@ -398,9 +402,53 @@ corresponding bits are set.
 The mechanisms for correlation are left to each implementation, and are
 considered to be beyond the scope of this specification.
 
-# Telemetry Report Formats { #report-format }
+## Flow Identification { #flow-identification }
+
+There is no explicit metadata defined for flow identification. The
+expectation is that either:
 
-This section specifies the packet formats for telemetry reports.
+  - a truncated packet fragment including the original packet headers will
+    be included in the telemetry report, allowing the distributed telemetry
+    monitoring system to categorize and identify flows in any manner that
+    it desires, or
+
+  - domain specific flow identification metadata will be included.
+
+Tunneled packets such as VXLAN packets raise the question whether flow
+identification should be based on outer or inner headers. The answer may
+vary depending on the goals of tracked flow monitoring, deployment aspects,
+operational issues, and the capabilities of the distributed telemetry
+monitoring system. Note that it is possible to identify flows based on
+inner packet headers even when using an INT encapsulation based on outer
+headers such as INT over TCP/UDP.
+
+When using INT-MD and flow identification based on inner headers is
+desired, the distributed telemetry monitoring system should parse the
+truncated packet fragment all the way down past any embedded telemetry
+metadata (if present), even when the Individual Report includes optional
+metadata such as drop reason. It may also want to process the embedded
+telemetry metadata, for example to recognize the case where a path change
+directs traffic to a congested node where packets are being dropped.
+
+## Coalescing A Group of Telemetry Reports In A Single Packet
+
+Starting with Version 2.0, the telemetry report format allows for a group
+of individual reports, each corresponding to one data plane packet, to be
```

Comment:
Can we reword it as 

of individual reports, each corresponding to one flow (multiple packets) instead of one data plane packet.

Reply:
I don't think I agree to that.

How about "one data plane packet or one flow"?

Reply:
I am ok with "one data plane packet or one flow".

Reply:
Done